spec/_attachments/ic.did

```diff
@@ -224,6 +231,18 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // Canister backup and restore
+  take_snapshot: (record { 
+    canister_id: canister_id;
+    replace_snapshot: opt snapshot_id;
+  }) -> (snapshot_id);
+  load_snapshot: (record { 
+    canister_id: canister_id;
+    snapshot_id: snapshot_id;
+  }) -> ();
+  list_snapshots: (record {canister_id : canister_id}) -> (vec snapshot);
+  delete_snapshot: (record {snapshot_id : snapshot_id}) -> ();
```

Comment:
In load_snapshot, we pass canister and snapshot ID while here we only pass snapshot ID. I believe the former is preferred, also for the sake of effective canister ID checks.